action.yml

```diff
@@ -0,0 +1,163 @@
+name: 'deploy-infrastructure'
+author: '1up-team'
+description: 'Action to plan and deploy Terraform modules'
+
+inputs:
+  module-name:
```

Comment:
module-name should be the service name not 'terraform'? I think we can extract service name and insert as default value if not provided

Reply:
I dont think so, we're in the domain of terraform here and the ubiquitous language is module.

Reply:
From what I understand, `module-name` is used to generate the terraform state file location, e.g. ecs_service_${module_name}. And, in most cases this is the service-name, we never had this as something like `ecs_service_terraform` but we have this as `ecs_service_freddy` or `ecs_service_brain`

Reply:
and service-name is not the best name too, agree, because it's not really the service name, it the state-file-key-prefix?